Dockerfile

```diff
@@ -47,21 +50,28 @@ ENV PATH "$PATH:/opt/bin"
 # Workaround for the transition to protoc-gen-go-grpc
 # https://grpc.io/docs/languages/go/quickstart/#regenerate-grpc-code
 WORKDIR /tmp
-RUN git clone -b v$grpc.x --recursive https://github.com/grpc/grpc-go.git
+RUN git clone -b "v${grpc}.x" --recursive https://github.com/grpc/grpc-go.git
 RUN ( cd ./grpc-go/cmd/protoc-gen-go-grpc && go install . )
 
 WORKDIR /tmp
-RUN git clone -b v$grpc_java.x --recursive https://github.com/grpc/grpc-java.git
+RUN git clone -b "v${grpc_java}.x" --recursive https://github.com/grpc/grpc-java.git
 WORKDIR /tmp/grpc-java/compiler
 RUN CXXFLAGS="-I/opt/include" LDFLAGS="-L/opt/lib" ../gradlew -PskipAndroid=true java_pluginExecutable
 
+# Add grpc-gateway & openapiv2 support (formerly known as protoc-gen-swagger)
+WORKDIR /tmp
```

Comment:
isn't this go-gettable? 

$ go install \
    github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway \
    github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2 \

Reply:
Yup.  I think we can install this with a single `go install` command.  I'll update the PR to match